utils/kmstest.cpp

```diff
@@ -700,21 +711,17 @@ static void print_outputs(const vector<OutputInfo>& outputs)
 			printf(" %s=%" PRIu64 ",", prop.prop->name().c_str(),
 			       prop.val);
 
-		printf("\n  Crtc %u/@%u", o.crtc->idx(), o.crtc->id());
+		printf("  Crtc %u/@%u", o.crtc->idx(), o.crtc->id());
```

Comment:
Linefeed change here... Looks odd, why would this patch require linefeed changes here?